scripts/demo.sh

```diff
@@ -3,10 +3,20 @@
 session="demo"
 window="ros2 vision demo"
 
-ros2_ws="~/ade-home/ros2_rolling/"
-package="ros2_shm_vision_demo"
-#set some video path
-video="./video/apex_ai.mp4"
+ros2_ws="$HOME/code/ros/rolling"
```

Comment:
@MatthiasKillat if you try these scripts locally, this line should be all that you have to change I think.